package.json

```diff
@@ -29,6 +29,7 @@
     "ionicons": "3.0.0",
     "rxjs": "5.5.2",
     "sw-toolbox": "3.6.0",
+    "tslint-config-airbnb": "^5.7.0",
```

Comment:
one thing that could my eye is to remove ^ from version. Just keep them fixed

Reply:
I've added a shrinkwrap file which effectively does the same thing